<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartmentGuiClass</class>
 <widget class="QWidget" name="DepartmentGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DepartmentGui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListWidget" name="volunteersList">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="emailLineEdit">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Interests</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="interestsLineEdit">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add Volunteer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="unassignedVolunteersList">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Most suitable volunteers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="assignButton">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Assign</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
